integration-tests/integration_test.go

```diff
@@ -308,7 +308,7 @@ func startStandaloneNode(t *testing.T, ctx context.Context, logMessages chan<- s
 	cmd.Env = append(
 		os.Environ(),
 		"VERBOSITY=5",
-		"DATA_DIR="+standaloneDataDir,
+		"LEVELDB_DATA_DIR="+standaloneDataDir,
```

Comment:
For `mesh` (not `mesh-bootstrap`) the environment variable is still called `DATA_DIR`.